pkg/flag/configs_test.go

```diff
@@ -0,0 +1,124 @@
+package flag
+
+import (
+	"encoding/json"
+	"strings"
+
+	apiv3 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"context"
+	"testing"
+)
+
+func TestMergeValues(t *testing.T) {
+
+	type args struct {
+		ctx context.Context
```

Comment:
If the `ctx` is not part of the test you can drop it and use a `context.Background` during the execution.